README.md

```diff
@@ -1,23 +1,34 @@
-# Responsive Type
+# Working with Responsive Type
 
 ## Overview
 
-In this lesson we will look at strategies for making adjustments to type in responsive layouts.
+CSS media queries are great for reshaping websites for any device, making
+them very responsive. There is still some work to do though, particularly
+in regard to content. In addition to media such as images, the text content also
+becomes a concern for readability. Depending on the screen size of a user's device, text can take up
+different amounts of space. On a site with static sized text, a user with a huge
+desktop monitor may see a website with large blank spaces and text sprawling in
+long lines across the screen, while a mobile user might be scrolling down
+through seemingly endless text.On a small screen, text can appear large,
```

Comment:
space

Reply:
...text with fixed size...

Reply:
@sgharms what is the revision being suggested for "text with fixed size"?